jenkins/parser/jenkins-parser-monitor-job.py

```diff
@@ -214,12 +214,16 @@
          <tr class="header">\n\
             <th>Blacklisted Node</th>\n\
             <th>Node Url</th>\n\
+            <th>Reason</th>\n\
          </tr>\n\
          </thead>\n\
       <tbody>\n'
     html_file.write(tail)
 
     for node in os.listdir("/var/lib/jenkins/workspace/cache/blacklist/"):
+        file_path = "/var/lib/jenkins/workspace/cache/blacklist/" + node
+        with open(file_path, "r") as file:
```

Comment:
@aandvalenzuela , may be you want to add a check if `file_path` exists or put this code under try/catch sot hat in case of file access/read error you can set reason="unknow"